server/cabal.project.local.fast

```diff
@@ -0,0 +1,17 @@
+---- PREVIEW OF CABAL-INSTALL WORKFLOW
+----   Try: 
+----     $ cabal new-build
+----   See: 
+----     https://github.com/hasura/graphql-engine/issues/3280 
+----     https://www.haskell.org/cabal/users-guide/nix-local-build-overview.html 
+
+-- Compile quickly without optimizations
+--
+-- TO ENABLE THIS YOU CAN DO: 
+--   $ ln -s cabal.project.local.prof cabal.project.local
+package *
+  documentation: true
+
+package graphql-engine
+  optimization: 0
```

Comment:
FWIW, this doesn’t actually work due to haskell/cabal#5353. The issue is that `optimization: 0` doesn’t override `-O2` in the `.cabal` file. We should probably remove `-O2` from the `.cabal` file and only put `optimization: 2` in the `cabal.project` file.

Reply:
Ya came here to post after just noticing this too. Trying to figure out wth build tools are doing is about my least favorite thing...

Reply:
But I guess this speaks to the benefit of encoding this knowledge in config files or comments at least

Reply:
A simple workaround, which I’ve been using, is to use `ghc-options: -O0` instead. But I think moving `-O2` out of the `.cabal` file is a better solution.

Reply:
(sorry for the delay). I updated the `cabal.project.local.fast` to mention the issue and work around it.